jetty-core/jetty-websocket/jetty-websocket-core-common/src/main/java/org/eclipse/jetty/websocket/core/util/WebSocketDemander.java

```diff
@@ -54,19 +56,25 @@ public abstract class DemandingFlusher extends IteratingCallback implements Dema
     /**
      * @param emitFrame where frames generated by {@link #handle(Frame, Callback, boolean)} are forwarded.
      */
-    public DemandingFlusher(IncomingFrames emitFrame)
+    public WebSocketDemander(IncomingFrames emitFrame)
     {
         _emitFrame = emitFrame;
     }
 
+    @Override
+    public void setNextDemand(DemandChain nextDemand)
```

Comment:
Should "Next" be "downstream"?  Or if next/prev makes sense, then use them instead of upstream/downstream ?